### DISCUSSION TITLE
Next.Js not updating after refreshed.

### DISCUSSION THREAD

**Discussion by jacobeugene40** [community/community#129181] [Programming Help]
### Body

What could possibly be the problem?

Any Next.Js developer in the house please? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Rudra-Sankha-Sinhamahapatra** [comment]
@jacobeugene40  Are you trying to hot reload?and are you running next js project inside a docker container?
If you are using docker then you have to use a technique called bind mounts ..you can read about it in the internet
But if you are running your local machine then there might be some error ..
please specify what kind of error you are facing...can you provide your code snippets

**Reply by jacobeugene40** [reply]
Okay sir, I will share the code snippets.

Thanks for your time.

**Reply by ebndev** [reply]
Hi @Rudra-Sankha-Sinhamahapatra, thanks for taking the time to answer and ask follow up questions! It's very helpful in GitHub Discussions.

@jacobeugene40 let us know if you have further questions by sharing the code snippet, as mentioned 😄
